src/form.tsx

```diff
@@ -16,7 +16,7 @@ if (
     import('scroll-behavior-polyfill').finally(() => {});
 }
 
-type FormValues = {[name: string]: any};
+export type FormValues = {[name: string]: any};
```

Comment:
you need to re-export it in `index.tsx` to make it public

Reply:
fixed